app/models/user.js

```diff
@@ -1,13 +1,20 @@
 module.exports = (sequelize, DataTypes) => {
   const User = sequelize.define(
-    'user',
+    'User',
```

Comment:
Why did you modified the name of model?

Reply:
Because in the bootstrap, when required, the model is renamed anyways, so I figured it will be better if I just name it using UpperCamelCase in the definition.

Reply:
You are right!